src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -54,14 +54,17 @@ protected string ParseSecretToken(ConfigurationKeyValue kv)
 		protected bool ParseCaptureHeaders(ConfigurationKeyValue kv)
 		{
 			if (kv == null || string.IsNullOrEmpty(kv.Value)) return true;
-			if (bool.TryParse(kv.Value, out var value)) return value;
 
-			return true;
+			return !bool.TryParse(kv.Value, out var value) || value;
```

Comment:
This was auto cleanup - if we revert back then the next auto cleanup will make it like this again.

Reply:
You might be using too aggressive option of Rider's code cleanup. I use `SilentCodeCleanUp`:
![image](https://user-images.githubusercontent.com/7782093/61325896-5d575300-a81e-11e9-9ce9-90991135eaeb.png)

Reply:
Yeah, I usually do full cleanup.

Plus to me it also suggests this change - I thought it's maybe based on our [`DotSettings` file](https://github.com/elastic/apm-agent-dotnet/blob/master/ElasticApmAgent.sln.DotSettings)  - but I'm not sure

<img width="634" alt="image" src="https://user-images.githubusercontent.com/1091853/61326456-174ac100-a817-11e9-9373-9842cd4d8759.png">

To me the green dots and the yellow thing was a little bit disturbing. Do you still want me to revert?

Reply:
Not a major thing for me - up to you. if you decide to revert, as I mentioned above, I would find it more useful to replace `true` in the code before the change with a named constant `DefaultValues.CaptureHeaders`.

Reply:
It would be nice to know why it's suggested - do you also see this suggestion? I think if it's defined in our `DotSettings` we should do it this way, if not I'd also not do this.

Reply:
Yes, I also see the suggestion. I copied the solution aside, deleted `.editorconfig` and I still see the same suggestion.

Reply:
Visual Studio also gives a suggestion, but it suggest an (in my opinion) more readable version.

<img width="1139" alt="image" src="https://user-images.githubusercontent.com/1091853/61327728-de601b80-a819-11e9-96b4-094c0639129e.png">

But once I take it, Rider still suggests the other one 🤦‍♂

Reply:
I tried to introduce a name constant `ConfigConsts.DefaultValues.CaptureHeaders` instead of `true` but Rider still suggest the same transformation because it sees the actual value of the constant:
```
return !bool.TryParse(kv.Value, out var value) || value;
```

 If I change `CaptureHeaders` to `static readonly` instead of `const` Rider still suggest to change `if` to `return` but at least it keeps `CaptureHeaders` in the picture
```
return bool.TryParse(kv.Value, out var value) ? value : ConfigConsts.DefaultValues.CaptureHeaders;
```

Which to me shows that using these suggestions automatically is too error prone and the benefit doesn't outweigh the risk. I frankly think that the first transformation is a bug because it takes the constant out of the picture so changing the constant after that will introduce a bug.

This particular change is not that important so the decision is yours but if I were you I would reconsider applying these changes automatically.